### PR TITLE
feat: add changelog for v3.13.0

### PR DESCRIPTION
**Pull request by shay2025**
## Description

This pull request introduces the changelog for the reviewpad v3.13.0 release.

## Checklist

- [x] I have performed a self-review of my code
- [x] I have run `yarn build` without any errors

## Code review and merge strategy (ship/show/ask)

- [x] Ask: this pull request requires a code review before merge

### CODE REVIEW

File: src/pages/changelog/reviewpad-v3130-faro-edition.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## New features :fire:
+
+- Get a [metrics report](/guides/syntax#mode) when the pull request is merged.
```
Comment (ferreiratiago) [inline src/pages/changelog/reviewpad-v3130-faro-edition.md:3]:
```suggestion
- Get the [metrics report](/guides/syntax#mode) when the pull request is merged.
```

File: src/pages/changelog/reviewpad-v3130-faro-edition.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+## Improvements :rocket: {#improvements}
+
+- Add closure reason to [`close`](/guides/built-ins#close).
```
Comment (ferreiratiago) [inline src/pages/changelog/reviewpad-v3130-faro-edition.md:7]:
```suggestion
- Add the parameter reason to [`close`](/guides/built-ins#close). Applicable only for issues.
```

**Review by ferreiratiago — CHANGES_REQUESTED** [review]
WDYT?

File: src/pages/changelog/reviewpad-v3130-faro-edition.md (hunk 2 above)
Comment (shay2025) [reply src/pages/changelog/reviewpad-v3130-faro-edition.md:7]: I don't think "parameter reason" makes much sence 😕

Comment (ferreiratiago) [reply src/pages/changelog/reviewpad-v3130-faro-edition.md:7]: Ok. Just append the `Applicable only for issues.`

**Pull request merged**